sparkles/core.py

```diff
@@ -1629,3 +1013,67 @@ def from_ocat(cls, obsid, t_ccd=-5, man_angle=5, date=None, roll=None, **kwargs)
         aca = get_aca_catalog(**params_proseco)
         acar = cls(aca)
         return acar
+
+
+check_acq_p2 = checks.acar_check_wrapper(checks.check_acq_p2)
+check_bad_stars = checks.acar_check_wrapper(checks.check_bad_stars)
+check_dither = checks.acar_check_wrapper(checks.check_dither)
+check_fid_count = checks.acar_check_wrapper(checks.check_fid_count)
+check_fid_spoiler_score = checks.acar_check_wrapper(checks.check_fid_spoiler_score)
+check_guide_count = checks.acar_check_wrapper(checks.check_guide_count)
+check_guide_fid_position_on_ccd = checks.acar_check_wrapper(
+    checks.check_guide_fid_position_on_ccd
+)
+check_guide_geometry = checks.acar_check_wrapper(checks.check_guide_geometry)
+check_guide_is_candidate = checks.acar_check_wrapper(checks.check_guide_is_candidate)
+check_guide_overlap = checks.acar_check_wrapper(checks.check_guide_overlap)
+check_imposters_guide = checks.acar_check_wrapper(checks.check_imposters_guide)
+check_include_exclude = checks.acar_check_wrapper(checks.check_include_exclude)
+check_pos_err_guide = checks.acar_check_wrapper(checks.check_pos_err_guide)
+check_too_bright_guide = checks.acar_check_wrapper(checks.check_too_bright_guide)
+
+
+def check_catalog(acar: ACACheckTable) -> None:
```

Comment:
It would make a little more sense to me to have check_catalog in checks, with an interface so that I could get the messages without writing back into the ACAReviewTable if desired.  But I'm not passionate about it.

Reply:
The idea was that if some other application wants to use the checks from `checks.py`, then that app has its own version of `check_catalog` that calls some or all of the `checks.py` checks. So the `check_catalog()` function in `sparkles.core` is specifically *only* for sparkles, but the checks in the `checks` module are somewhat public and modular.

Of course another application would be free to copy the current `check_catalog` with essentially no modification except the last few lines.